docs/ref/request-response.txt

```diff
@@ -998,3 +998,20 @@ Attributes
 .. attribute:: StreamingHttpResponse.streaming
 
     This is always ``True``.
+
+FileResponse objects
+====================
+
+.. versionadded:: 1.8
+
+.. class:: FileResponse
+
+:class:`FileResponse` is a subclass of :class:`StreamingHttpResponse` optimized
+for binary files. It uses ``wsgi.file_wrapper``_ when available.
```

Comment:
I think it would help to clarify if there any benefits when `wsgi.file_wrapper` isn't available -- and, without trying to provide an exhaustive list, also add some details about generally when `file_wrapper` is and isn't available.